api/internal/impl/api/acl/create.go

```diff
@@ -65,6 +69,7 @@ import (
 func CreatePolicy(source *workloadapi.X509Source,
 	name string, SPIFFEIDPattern string, pathPattern string,
 	permissions []data.PolicyPermission,
+	allow predicate.Predicate,
```

Comment:
Added `predicate` parameter to `impl` methods.

These should not affect the publicly-facing API since they are all used internally.